tensorflow/core/protobuf/autotuning.proto

```diff
@@ -43,9 +43,10 @@ message AutotuneResult {
     oneof key {
       ConvKey reference_conv = 11;
       GemmKey reference_gemm = 12;
+      CudaConvPlanKey reference_cuda_conv_plan = 13;
     }
 
-    int64 buffer_address = 13;
+    int64 buffer_address = 14;
```

Comment:
Can you not change ids of other fields? Ids don't have to be increasing in textual order.

Reply:
Sure, I just renumbered those fields. But I am still not clear what is the basic rule for the numbering. The strategy I use now is to number them from the value shown in the comment of `// // Next ID: 14`.